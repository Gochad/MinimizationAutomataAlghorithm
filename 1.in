A B C
A
C
0 1
A 0 A
A 1 A
A 0 B
B 1 C
