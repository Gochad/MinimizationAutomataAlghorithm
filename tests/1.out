A AB AC 
A
AC 
0 1 
A 0 AB
A 1 A
AB 0 AB
AB 1 AC
AC 0 AB
AC 1 A
